camligorkem/assignment3/Dictionary.java

```diff
@@ -0,0 +1,85 @@
+// Trie Node class to create Dictionary based on prefixes
+class TrieNode{
+    TrieNode[] letters;
```

Comment:
Maybe better use a map here, this way you won't have to store space for 26 children for every node that you create.